.github/settings.yml

```diff
@@ -0,0 +1,98 @@
+---
+repository:
+  name: ocis-accounts
+  description: ':atom_symbol: :busts_in_silhouette: Serve Accounts for oCIS'
```

Comment:
Two emojis?

Reply:
too ugly? :p

Reply:
All repos only got one so far, right? :)

Reply:
You're right, sticking to one :)